.doc_gen/metadata/services.yaml

```diff
@@ -50,6 +60,16 @@ cognito-identity:
   short: "&COGID;"
   sort: "Cognito Identity"
   version: cognito-identity-2014-06-30
+cognito-idp:
+  long: "&COGUP;"
+  short: "&COGUP;"
```

Comment:
These entities are for user pools, not identity provider. I think we need another custom entity for this:
```
<!ENTITY COGIDP "&COGID; Provider">
```

Reply:
I'll add another entity in a new PR.

Reply:
Fixed.